apps/avatax/scripts/migrations/update-webhooks.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+
+import { AuthData } from "@saleor/app-sdk/APL";
+import { webhookMigrationRunner } from "@saleor/webhook-utils";
+import { createInstrumentedGraphqlClient } from "../../src/lib/create-instrumented-graphql-client";
+import { appWebhooks } from "../../webhooks";
+
+export const updateWebhooksScript = async ({
+  authData,
+  dryRun,
+}: {
+  authData: AuthData;
+  dryRun: boolean;
+}) => {
+  console.log("Working on env: ", authData.saleorApiUrl);
+
+  const client = createInstrumentedGraphqlClient({
+    saleorApiUrl: authData.saleorApiUrl,
+    token: authData.token,
+  });
+
+  await webhookMigrationRunner({
+    client,
+    dryRun,
+    getManifests: async ({ appDetails }) => {
+      const webhooks = appDetails.webhooks;
+
+      if (!webhooks?.length) {
+        console.info("The environment does not have any webhooks, skipping");
+        return [];
+      }
+
+      // All webhooks in this application are turned on or off. If any of them is enabled, we enable all of them.
```

Comment:
I dont get this comment here, it should be put in L44
Also I don't know why it happens